README.md

```diff
@@ -76,6 +76,7 @@ This image has everything you need to try out PostHog locally! It will set up a
 9. Run migrations `DEBUG=1 python3 manage.py migrate`
 10. Run `DEBUG=1 python3 manage.py runserver`
 11. Run the tests and frontend
+12. Start celery to process background jobs `DEBUG=1 celery -A posthog worker --loglevel=info`
```

Comment:
Maybe worth creating a little `bin` script that starts both, maybe even with the frontend too at some point

Reply:
sure, that's done here: https://github.com/PostHog/posthog/pull/473/commits/10fb8e8f1bfc3e48df8d98115a744d750ee732d1